pytest_plugins/manual_skipped.py

```diff
@@ -9,33 +9,56 @@
 
 def pytest_configure(config):
     """Register custom marker for stubbed test cases."""
-    marker = "stubbed: Tests that are not automated yet or manual only."
-    config.addinivalue_line("markers", marker)
+    config.addinivalue_line('markers', 'stubbed: Tests that are not automated yet or manual only.')
 
 
 def pytest_collection_modifyitems(session, config, items):
-    """
-    Collects tests based on pytest option to select tests marked as failed on Report Portal
-    """
-    mark_passed = config.getvalue('mark_manuals_passed')
-    mark_skipped = config.getvalue('mark_manuals_skipped')
-    if mark_passed or (not mark_skipped):
-        return
+    """Remove/Include stubbed tests from collection based on CLI option"""
+    opt_passed = config.getvalue('mark_manuals_passed')
+    opt_skipped = config.getvalue('mark_manuals_skipped')
+    # TODO turn this into a flag or a choice option, this logic is just silly.
+    mark_skipped = opt_skipped and (not opt_passed)
+    include_stubbed = config.getvalue('include_stubbed')
```

Comment:
I think it would be clearer if all the options were named consistently with `manuals` or `stubbed`, but not a mix of both. Otherwise, LGTM.

Reply:
100% agree, tried to keep the scope limited though. When the TODO is addressed we can change the name to `--mark-stubbed [skipped|pass]`